### DISCUSSION TITLE
Missing translation?

### DISCUSSION THREAD

**Discussion by Mister-Hope** [vuejs-translations/docs-zh-cn#381] [Q&A] [answered]
![Screenshot_2022-05-25-11-54-37-85_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/33315834/170176385-59018f56-36d1-4057-977b-61cd7fed21d2.jpg)

这里没翻译，我觉得是应该把第二个单词翻译一下的，它不是后文的API名称，应该翻译也不需要保留英文。

如果的确遗漏你们可以顺手修改一下，或者也可以叫我一下我发个pr也可以。

（在高铁上太无聊了

**Comment by wxsms** [comment, marked as answer]
根据[以往的共识](https://github.com/vuejs/docs-next-zh-cn/issues/613)，prop drilling 翻译为“prop 逐级透传”，欢迎贡献 PR。
